sample.env

```diff
@@ -103,4 +103,8 @@ AEM_LOG_TO_FILE=false
 AEM_LOG_LEVEL=INFO
 ADM_LOG_TO_FILE=false
 ADM_LOG_TO_CONSOLE=true
-ADM_LOG_LEVEL=INFO
\ No newline at end of file
+ADM_LOG_LEVEL=INFO
+
+# ODE Monitoring
+ODE_TIM_INGEST_MONITORING_ENABLED=false
+TIM_INGEST_MONITORING_INTERVAL=
```

Comment:
Should we enable TIM ingest monitoring by default?

Reply:
I went back and forth on setting it to enabled by default. I'm not attached to either decision, but I chose to disable it by default since it's not required for the software to run - it's just a nice to have for certain scenarios. I'm open to your thoughts, as usual!

Reply:
In most cases, we’ll likely want to know if the ODE isn’t receiving TIM deposits. Since this doesn’t significantly affect the ODE's behavior, it makes sense to me to enable the feature by default, allowing operators to disable it if they prefer not to use it.